playwright-e2e/pages/dsm/participants/participants-page.ts

```diff
@@ -51,4 +51,14 @@ export default class ParticipantsPage {
     await input.toLocator().type(searchString);
     await Promise.all([this.page.locator('.fa-spinner').waitFor({ state: 'visible' }), this.searchButton().click()]);
   }
+
+  async filterMedicalRecordPts(): Promise<void> {
+    await this.page.locator("text=Customize View >> button").click();
```

Comment:
Not sure if you already know this, with text selector in Playwright, string without quotes can match more elements. Do you want to use quotes to search for exact match?

Reply:
That was my intention, do you think it would be better if I was not searching for an exact match here? I am still somewhat knew to the standards for playwright.